scripts/TestHarness/testers/RavenErrors.py

```diff
@@ -11,103 +11,129 @@
-    else:
-      return ' '.join(["python",raven,ravenflag,self.specs["input"]])
+    if self.specs['test_interface_only'].lower() == 'true':
+      ravenflag = 'interfaceCheck '
+    if RavenUtils.in_python_3():
+      return ' '.join(["python3", raven, ravenflag, self.specs["input"]])
+    return ' '.join(["python", raven, ravenflag, self.specs["input"]])
 
 
   def __init__(self, name, params):
+    """
+      Initializer for the class. Takes a String name and a dictionary params
+      @ In, name, string, name of the test.
+      @ In, params, dictionary, parameters for the class
+      @ Out, None.
+    """
     Tester.__init__(self, name, params)
-    self.required_libraries = self.specs['required_libraries'].split(' ')  if len(self.specs['required_libraries']) > 0 else []
+    self.required_libraries = self.specs['required_libraries'].split(' ')  \
+      if len(self.specs['required_libraries']) > 0 else []
     self.required_executable = self.specs['required_executable']
-    self.required_executable = self.required_executable.replace("%METHOD%",os.environ.get("METHOD","opt"))
+    self.required_executable = self.required_executable.replace("%METHOD%",
+                                                                os.environ.get("METHOD", "opt"))
     self.specs['scale_refine'] = False
 
-  def checkRunnable(self, option):
-    """This method checks if the the test is runnable within the current settings"""
-    missing,too_old,notQA = RavenUtils.checkForMissingModules()
+  def check_runnable(self):
+    """
+      This method checks if the the test is runnable within the current settings
+      @ In, None
+      @ Out, check_runnable, boolean, If True this test can run.
+    """
+    missing, too_old, _ = RavenUtils.check_for_missing_modules()
     if len(missing) > 0:
-      self.setStatus('skipped (Missing python modules: '+" ".join(missing)+
-                     " PYTHONPATH="+os.environ.get("PYTHONPATH","")+')',
-                     self.bucket_skip)
+      self.set_status('skipped (Missing python modules: '+" ".join(missing)+
+                      " PYTHONPATH="+os.environ.get("PYTHONPATH", "")+')',
+                      self.bucket_skip)
       return False
     if len(too_old) > 0:
-      self.setStatus('skipped (Old version python modules: '+" ".join(too_old)+
-                     " PYTHONPATH="+os.environ.get("PYTHONPATH","")+')',
-                     self.bucket_skip)
+      self.set_status('skipped (Old version python modules: '+" ".join(too_old)+
+                      " PYTHONPATH="+os.environ.get("PYTHONPATH", "")+')',
+                      self.bucket_skip)
       return False
     for lib in self.required_libraries:
       if platform.system() == 'Windows':
         lib += '.pyd'
       else:
         lib += '.so'
       if not os.path.exists(lib):
-        self.setStatus('skipped (Missing library: "'+lib+'")', self.bucket_skip)
+        self.set_status('skipped (Missing library: "'+lib+'")', self.bucket_skip)
```

Comment:
@joshua-cogliati-inl this is very similar to the other testers...why do we need this approach?

Reply:
changed to set_skip